index.html

```diff
@@ -0,0 +1,751 @@
+                <li class="list-item"><a class="btn-link" href="#sec-input">Input</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-list">List</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-navigation">Navigation</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-modal">Modal</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-rating">Rating</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-slider">Slider</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-grid">Grid</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-text-utilities">Text Utilities</a></li>
+                <li class="list-item"><a class="btn-link" href="#sec-toast">Toast</a></li>
+            </ul>
+        </aside>
+        <div class="lib-container">
+            <!-- Get Started -->
+            <section id="sec-start">
+                <h2 class="txt-header-2">Get <span class="txt-header-inv">Started</span></h2>
+                <p class="txt-desc">Add this library by including the following CSS and JavaScript files from CDN</p>
+                <br />
+                <h3 class="txt-header-3">CSS</h3>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%250Ahref%253D%2522https%253A%252F%252Fstylesup.netlify.app%252Fstyles%252Fcomponents.css%2522%253E"
+                        style="width: 639px; height: 223px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <h3 class="txt-header-3">Javascript</h3>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cscript%2520type%253D%2522text%252Fjavascript%2522%250Asrc%253D%2522https%253A%252F%252Fstylesup.netlify.app%252Fscripts%252Fscript.js%2522%253E%250A%253C%252Fscript%253E"
+                        style="width: 597px; height: 242px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+            <!-- Avatar -->
+            <section id="sec-avatar">
+                <h2 class="txt-header-2 txt-header-inv">Avatar</h2>
```

Comment:
text-header-inv name is not clear. Can you use some descriptive name here ?

Reply:
sure, may be i'll make this class generic and use it across with the name txt-color-inverse. Does that sound good?